### PR TITLE
Review main

### PR DESCRIPTION
**Pull request by andre-richter**
[^1] Should result in a footnote on gitbooks.

### CODE REVIEW

File: src/main.md [hunk 1]
```diff
@@ -295,9 +295,12 @@ memory (Flash); the LMA is where in Flash those initial values are stored.
 Finally, we associate a symbol to the LMA of `.data`.
 
 On the Rust side, we zero the `.bss` section and initialize the `.data` section. We can reference
-the symbols we created in the linker script from the Rust code. The *addresses* of these symbols are
+the symbols we created in the linker script from the Rust code. The *addresses*[^1] of these symbols are
 the boundaries of the `.bss` and `.data` sections.
 
+[^1]: The fact that the addresses of the linker script symbols must be used here can be confusing and
```
Comment (japaric) [inline src/main.md]: This does get render as a footnote by mdbook but it's not automatically placed at the end of the page (see image below). You'll have to move it to the bottom manually.

---

**Comment by andre-richter** [comment]
I updated that. For convenience I use the website for these reviews. Feel free to squash the commits.

**Comment by japaric** [comment]
Thanks!

bors r+

**Pull request merged**